spec/requests/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns http success for users#index action' do
+      get users_url
+      expect(response).to have_http_status(:success) # 200
+      expect(response.body).to include('Here is a list of users')
+    end
```

Comment:
- [ ] Please make sure to add the below test for the user index action as requested in the project requirements.

![image](https://github.com/binyamolango/Blog-App2/assets/83803414/cfe2fac3-e1e5-47b5-9215-862518548804)